docs/versioned_docs/version-4.0.x/user-guides/replicated-storage-user-guide/replicated-pv-mayastor/platform-support.md

```diff
@@ -19,20 +19,20 @@ To install Replicated PV Mayastor using Helm on MicroK8s, execute the following
 **Command**
 
 ```
-helm install mayastor mayastor/mayastor -n mayastor --create-namespace  --set csi.node.kubeletDir="/var/snap/microk8s/common/var/lib/kubelet"
+helm install mayastor mayastor/mayastor -n openebs --create-namespace  --set csi.node.kubeletDir="/var/snap/microk8s/common/var/lib/kubelet"
```

Comment:
This is installing the mayastor chart directly?

Reply:
Done.